### ISSUE TITLE
data-value attribute is not being tracked

### ISSUE THREAD

**Issue by ceremcem** [loadingio/loading-bar#10]
Changing `data-value` attribute dynamically by the template engine (RactiveJS, in my case) does not make progress bar to update its value. 

Tip: Re-rendering the progress bar solves the issue.

**Comment by Kansuler** [comment]
You are supposed to use the set method of the loading-bar object, from what i understand. You can react on prop change and trigger the set method instead.

**Comment by ceremcem** [comment]
Correct, that's [what I do](https://github.com/aktos-io/scada.js/blob/294cbbf029f8111b487dd15ceb3bf07c0c823259/components/progress/index.ls#L68-L76) at the moment. I kept the issue open and left the decision to the owner to clarify if current one is the supposed behavior or if the loading-bar object should observe its `data-value` attribute.

**Comment by abhipatel7** [comment]
@ceremcem Can you share your implementation? I'm having the same problem and am not able to figure out how I should fix it.
